src/components/container/ConnectDexcom/ConnectDexcom.stories.tsx

```diff
@@ -0,0 +1,59 @@
+﻿import React from 'react'
+import ConnectDexcom, { ConnectDexcomProps } from './ConnectDexcom'
+import { Card, Layout } from '../../presentational'
+
+export default {
```

Comment:
Probably not a huge deal, but I think the way these stories are set up (which is based on a bunch of other stories) is not the Typescript way that is suggested by Storybook (https://storybook.js.org/docs/writing-stories/typescript#typing-stories-with-meta-and-storyobj).  I'd been trying to switch over to their suggested pattern because it provides a little more type safety and I think actually helps with errors and updating stories if the properties of a Component change.

Reply:
Sounds reasonable to me.  I've converted to the suggested format.